src/test/java/apicurito/tests/TestRunner.java

```diff
@@ -1,26 +1,34 @@
 package apicurito.tests;
 
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
-import org.junit.runner.RunWith;
-
-import com.codeborne.selenide.Configuration;
-
 import apicurito.tests.configuration.Component;
 import apicurito.tests.configuration.TestConfiguration;
 import apicurito.tests.configuration.templates.ApicuritoTemplate;
+import apicurito.tests.steps.CommonSteps;
 import apicurito.tests.utils.openshift.OpenShiftUtils;
+import com.codeborne.selenide.Configuration;
 import cucumber.api.CucumberOptions;
 import cucumber.api.junit.Cucumber;
+import io.syndesis.qe.marketplace.openshift.OpenShiftConfiguration;
+import io.syndesis.qe.marketplace.openshift.OpenShiftService;
+import io.syndesis.qe.marketplace.openshift.OpenShiftUser;
+import io.syndesis.qe.marketplace.quay.QuayService;
+import io.syndesis.qe.marketplace.quay.QuayUser;
```

Comment:
are these marketplace imports necessary here?

Reply:
Definitely no. I forgot to remove them.

Reply:
fixed